activesupport/test/transliterate_test.rb

```diff
@@ -58,6 +58,15 @@ def test_transliterate_handles_unknown_object
     assert_equal "Can only transliterate strings. Received Object", exception.message
   end
 
+  def test_transliterate_returns_early_if_ascii_only
+    string = Minitest::Mock.new
+    string.expect(:is_a?, true, [String])
+    string.expect(:encoding, Encoding::UTF_8)
+    string.expect(:ascii_only?, true)
+    ActiveSupport::Inflector.transliterate(string)
+    string.verify
+  end
```

Comment:
I don't think it makes sense to add a test that asserts a specific implementation like this.

Reply:
I also wasn't sure if it made sense as there aren't really any assertions. It does help prevent the accidental removal of the optimization.

Let me know

Reply:
Yeah, please remove that test.